src/overlays/actors/ovl_En_Dt/z_en_dt.c

```diff
@@ -28,59 +95,714 @@ ActorInit En_Dt_InitVars = {
+    /* 07 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2AC3] "If the soldiers wish to run, then abandon us"   */
+    /* 08 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2AC4] "Let's not bring my wife into this" */
+
+    /* 09 */ EN_DT_CS_FOCUS_TARGET_BAISEN, /* [2ABB] "All must take refuge!!!" */
+    /* 10 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2ABC] "On with the carnival!!!" */
+
+    /* 11 */ EN_DT_CS_FOCUS_TARGET_BAISEN, /* [2AC6] "Ah!" */
+    /* 12 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2AC7] "Ah!" */
+    /* 13 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2AC8] "Ah!" */
+    /* 14 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2AC9] "The Couple's Mask. Someone got married" */
+    /* 15 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2ACA] "I wonder if my wife fled" */
+    /* 16 */ EN_DT_CS_FOCUS_TARGET_BAISEN, /* [2ACB] "What did you just say!?" */
+    /* 17 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2ACB] "We're all worried, let's end the meeting" */
+    /* 18 */ EN_DT_CS_FOCUS_TARGET_BAISEN, /* [2ACD] "But Mayor!" */
+    /* 19 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2ACE] "Stay or run, that's up to people to decide" */
+    /* 20 */ EN_DT_CS_FOCUS_TARGET_MUTO,   /* [2ACF] "..!" */
+    /* 21 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2AD0] "Thanks for helping me end that. Here's your reward" */
+    /* 22 */ EN_DT_CS_FOCUS_TARGET_MAYOR,  /* [2AD1] "I'm an unreliable mayor, but I want to protect my family" */
+
+    /* 23 */ EN_DT_CS_FOCUS_TARGET_MAYOR /* [2AC5] "You take refuge, too." */
+};
+
+typedef struct {
+    s16 D_80BEB268[1];
+    s16 D_80BEB26A[25];
+} CutsceneIndexTable;
+
+static CutsceneIndexTable CsIndexTable = { { 0x2ABB },
+                                           { 0x0000, 0x2ABD, 0x0000, 0x2AC0, 0x0001, 0x2AC1, 0x0002, 0x2AC4, 0x0003,
+                                             0x2AC6, 0x0004, 0x2AC7, 0x0005, 0x2AC8, 0x0006, 0x2AC9, 0x0007, 0x2ACC,
+                                             0x0008, 0x2ACF, 0x0009, 0x2AD0, 0x000A, 0x0309, 0x0309 } };
+
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 25, 70, 0, { 0, 0, 0 } },
 };
 
-#endif
+static AnimationHeader* sAnimations[EN_DT_ANIMATION_MAX] = { &object_dt_Anim_00112C, &object_dt_Anim_0005A4,
+                                                             &object_dt_Anim_000854, &object_dt_Anim_000DA8,
+                                                             &object_dt_Anim_000BE0, &object_dt_Anim_00B500 };
+
+static u8 sAnimationModes[EN_DT_ANIMATION_MAX] = { ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_ONCE,
+                                                   ANIMMODE_LOOP, ANIMMODE_LOOP, ANIMMODE_ONCE };
+
+// clang-format off
+static s32 sVisualStateTable[] = {
```

Comment:
I think this table are the Eye Textures and needs to be adjusted to be the right format.